src/Modules/CalcPerform.lua

```diff
@@ -526,15 +526,19 @@ local function doActorAttribsPoolsConditions(env, actor)
 				end
 			end
 			
-      local stats = { output.Str, output.Dex, output.Int }
-      table.sort(stats)
-      output.LowestAttribute = stats[1]
-      condList["TwoHighestAttributesEqual"] = stats[2] == stats[3]
-      
+			local stats = { output.Str, output.Dex, output.Int }
+			table.sort(stats)
+			output.LowestAttribute = stats[1]
+			condList["TwoHighestAttributesEqual"] = stats[2] == stats[3]
+		
 			condList["DexHigherThanInt"] = output.Dex > output.Int
 			condList["StrHigherThanDex"] = output.Str > output.Dex
 			condList["IntHigherThanStr"] = output.Int > output.Str
 			condList["StrHigherThanInt"] = output.Str > output.Int
+
+			condList["StrHighestAttribute"] = output.Str >= output.Dex  or output.Str >= output.Int
```

Comment:
Relatively new to this so apologies if I'm wrong but should these conditions be `and` instead of `or`?

Reply:
It should be "and" I just fixed it the last commit. https://github.com/PathOfBuildingCommunity/PathOfBuilding/pull/5185/commits/16910458d327a65369be1fed975aca5bd046136d